packages/cli/src/cli/services/commands/version.ts

```diff
@@ -1,15 +1,6 @@
 import {CLI_KIT_VERSION} from '@shopify/cli-kit/common/version'
-import {checkForNewVersion, packageManagerFromUserAgent} from '@shopify/cli-kit/node/node-package-manager'
-import {outputInfo, outputContent, outputToken, getOutputUpdateCLIReminder} from '@shopify/cli-kit/node/output'
+import {outputInfo, outputContent, outputToken} from '@shopify/cli-kit/node/output'
 
 export async function versionService(): Promise<void> {
-  const cliDependency = '@shopify/cli'
-  const currentVersion = CLI_KIT_VERSION
-  outputInfo(outputContent`Current Shopify CLI version: ${outputToken.yellow(currentVersion)}`.value)
-  // Check a max of once daily
-  const lastVersion = await checkForNewVersion(cliDependency, currentVersion, {cacheExpiryInHours: 24})
-  if (lastVersion) {
-    const packageManager = packageManagerFromUserAgent()
-    outputInfo(getOutputUpdateCLIReminder(packageManager, lastVersion))
-  }
+  outputInfo(outputContent`Current Shopify CLI version: ${outputToken.yellow(CLI_KIT_VERSION)}`.value)
```

Comment:
I made some further adjustments, so the output string isn't colored in any way. That should make it work smoothly in scripts that might call `shopify version` and do something with that information.

I also updated the warning to use stderr instead of stdout, so that commands intended for piping elsewhere won't have the warning output getting in their way.

![Screenshot 2024-08-12 at 13 44 14](https://github.com/user-attachments/assets/ca3d5c3d-0484-4cd4-bf4c-0842625f6fc3)